configs/samples/asterisk.conf.sample

```diff
@@ -126,6 +126,10 @@ documentation_language = en_US	; Set the language you want documentation
                 ; housekeeping AMI and ARI channel events.  This can
                 ; reduce the load on the manager and ARI applications
                 ; when the Digium Phone Module for Asterisk is in use.
+;sounds_search_custom_dir = no;  This option, if enabled, will
```

Comment:
It's explained in the description, but isn't the option name a bit misleading, by implying that the custom dir won't be searched at all, if it's set to no? Maybe `sounds_search_custom_dir_first` would be better? Alternately `sounds_prefer_custom_dir` to make it clear it's just an ordering preference, not yes/no altogether.

Reply:
It _won't_ be searched at all if set to "no".

Reply:
Okay, I was confused because this phrasing implies it does: set to "true", asterisk will search the custom directory for sounds files *before* searching the standard directory*

Should that be "instead of"?